oedi/config.yaml

```diff
@@ -34,6 +34,13 @@ AWS:
         - s3://oedi-data-lake/ATB/electricity/parquet/2021/
         - s3://oedi-data-lake/ATB/electricity/parquet/2022/
         - s3://oedi-data-lake/ATB/electricity/parquet/2023/
+    - Identifier: atb_transportation
+      Name: oedi_atb_transportation
+        Locations:
+          - s3://oedi-data-lake/ATB/transportation/parquet/2022/fuels_marine
+          - s3://oedi-data-lake/ATB/transportation/parquet/2022/fuels
+          - s3://oedi-data-lake/ATB/transportation/parquet/2022/vehicles
+          - s3://oedi-data-lake/ATB/transportation/parquet/2022/vehicles_fuels
```

Comment:
Just wondering, instead of adding `atb_transportation`, would we consider add `Locations` under `atb` directly? as they are all ATB datasets. 

No tech issue as you put here, boths ways would work, a simple check your consideration of putting this way.